src/components/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+
+type ButtonProps = {
+  name: string;
+  content: string;
+  handleFilter: (action: string) => void;
+  filter: string;
+  isReversed: boolean;
+};
+
+export const Button: React.FC<ButtonProps> = ({
+  name,
+  content,
+  handleFilter,
+  filter,
+  isReversed,
+}) => {
+  const buttonClass = classNames('button', {
+    'is-info': name === 'alphabet',
+    'is-success': name === 'length',
+    'is-warning': name === 'reverse',
+    'is-danger': name === 'reset',
```

Comment:
looks like better to create some helper function that will accept the name and return the className, you can use the switch/case there